symengine/tests/ntheory/test_ntheory.cpp

```diff
@@ -580,7 +584,10 @@ TEST_CASE("test_nthroot_mod(): ntheory", "[ntheory]")
     nthroot_mod_list(roots, i9, i2, i27);
     REQUIRE(roots.size() == 6);
     v = {integer(3), integer(6), integer(12), integer(15), integer(21), integer(24)};
-    bool same = std::equal(v.begin(), v.end(), roots.begin(), SymEngine::eq);
+    bool same = std::equal(v.begin(), v.end(), roots.begin(),
+            [](const RCP<const Basic> &a, const RCP<const Basic> &b) {
```

Comment:
That works! I thought if you add (), that it calls the `operator()`, so I don't understand how it works, but it compiles and tests run.

My wifi doesn't allow me to push in changes, but I have it in a branch and push in later.